cli/pcluster/easyconfig.py

```diff
@@ -383,7 +373,86 @@ def scheduler_handler(scheduler):
         scheduler_info["max_size"] = "max_vcpus"
         scheduler_info["initial_size"] = "desired_vcpus"
     else:
-        scheduler_info["parameters_to_remove"] = "max_vcpus", "desired_vcpus", "min_vcpus", "compute_instance_type"
+        scheduler_info["parameters_to_remove"] = ("max_vcpus", "desired_vcpus", "min_vcpus", "compute_instance_type")
         scheduler_info["max_size"] = "max_queue_size"
         scheduler_info["initial_size"] = "initial_queue_size"
     return scheduler_info
+
+
+def _prompt_a_list(message, options, default_value=None):
+    """
+    Wrap prompt to use it for list.
+
+    :param message: the message to show the user
+    :param options: the list of item to show the user
+    :param default_value: the default value
+    :return: the validate value
+    """
+    if not options:
+        print("ERROR: No options found for {0}".format(message))
+        sys.exit(1)
+    if not default_value:
+        default_value = options[0]
+
+    numbered_items = NumberedList(options)
+
+    def input_to_parameter(to_transform):
+        try:
+            item = numbered_items.get_item_by_index(int(to_transform))
+        except ValueError:
+            item = to_transform
+        return item
+
+    return prompt(
+        message,
+        validator=lambda x: x in options,
```

Comment:
same here. Users usually have restricted IAM roles that might allow them to use a resource but not list all. I think we should accept textual input or print an option such as `0. enter manually`